### ISSUE TITLE
Zevenet-5.10.1 supressing "Transfer-Encoding: chunked" header

### ISSUE THREAD

**Issue by jschyra** [zevenet/zproxy#3]
When I tested the load balancer with chunked encoding, I noticed that the HTTP header "Transfer-Encoding: chunked" is removed in the communication between the load balancer and the server
Unfortunately this causes a HTTP/500 error on my backend.

Communication client - lb:
```
POST /test HTTP/1.1
Content-Type: text/xml; charset=UTF-8
SOAPAction: "urn:test"
User-Agent: Axis2
Accept-Encoding: gzip
Content-Encoding: gzip
Host: lbname
Cookie: $Version=0; JSESSIONID=167EC628F488A461679844EAC9D6D1DA; $Path=/
Transfer-Encoding: chunked

1e1
[REQUEST_BODY]
0

HTTP/1.1 500 Internal Server Error
Content-Type: text/xml;charset=UTF-8
Transfer-Encoding: chunked
Date: Tue, 24 Mar 2020 09:07:49 GMT
Connection: close
Server: servername

19e
<?xml version='1.0' encoding='UTF-8'?><soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"><soapenv:Header xmlns:wsa="http://www.w3.org/2005/08/addressing"><wsa:Action>http://www.w3.org/2005/08/addressing/soap/fault</wsa:Action></soapenv:Header><soapenv:Body><soapenv:Fault><faultcode></faultcode><faultstring>unknown</faultstring><detail /></soapenv:Fault></soapenv:Body></soapenv:Envelope>
0


```

Communication lb - server:
```
POST /test HTTP/1.1
Content-Type: text/xml; charset=UTF-8
SOAPAction: "urn:test"
User-Agent: Axis2
Accept-Encoding: gzip
Content-Encoding: gzip
Host: lbname
Cookie: $Version=0; JSESSIONID=167EC628F488A461679844EAC9D6D1DA; $Path=/
X-Forwarded-For: 10.22.37.126

1e1
[REQUEST_BODY]
0

HTTP/1.1 500 Internal Server Error
Content-Type: text/xml;charset=UTF-8
Transfer-Encoding: chunked
Date: Tue, 24 Mar 2020 09:07:49 GMT
Connection: close
Server: servername

19e
<?xml version='1.0' encoding='UTF-8'?><soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"><soapenv:Header xmlns:wsa="http://www.w3.org/2005/08/addressing"><wsa:Action>http://www.w3.org/2005/08/addressing/soap/fault</wsa:Action></soapenv:Header><soapenv:Body><soapenv:Fault><faultcode></faultcode><faultstring>unknown</faultstring><detail /></soapenv:Fault></soapenv:Body></soapenv:Envelope>
0


```

**Comment by abdessamad-zevenet** [comment]
I'm glad tha it worked, the patch will be included in the next release once testing is over. Thank you for reporting the issue.
Regards.

**Comment by jschyra** [comment]
Sorry for annoing you but may you please so kind to tell me when you plan to release the new version?
Thanks in advanced!

**Comment by emiliocampos-zevenet** [comment]
Hi, we hope to release a new version along the following week, keep you posted.

Thanks

**Comment by abdessamad-zevenet** [comment]
Issue fixed in version v0.2.0
